### ISSUE TITLE
Deeplink to public chat on onboarding

### ISSUE THREAD

**Issue by hesterbruikman** [status-im/status-react#11236]
# Problem
One of the biggest problems we have is user activation. 

Influencers: The influencers have to spend time explaining to their audience/followers how to join their public chats within Status. This creates friction and a relevant % of users likely don’t bother doing it and drop off.

### User story

* [ ] As a new user invited to a public chat on Status (with or without a referral code), I want to join this public chat upon onboarding so I instantly have access to people who can help me navigate and discover the app
* [ ] As Status core contributor (Marketing) I want to create a link for an influencer that directs users to a public chat, because they might drop off if they have to search for the chat
* [ ] As a user joining Status I don't want for my IP address to be exposed without any action from my side that controls this
* [ ] Add public chat to list view when passed through deep link (with/without referral code)
* [ ] Show empty public chat view with option to join when user taps on public chat in list
* [ ] Fetch history from history node only after user taps Join in public chat view

**Comment by cammellos** [comment]
@hesterbruikman @errorists are you going to provide designs for this issue?
thanks!

**Comment by errorists** [comment]
[Design reusing parts already found in Group Chats](https://www.figma.com/file/aS1ct66VQ6V0cio7vSqS8UoG/Chat?node-id=17439%3A25). @hesterbruikman we need something for the grey copy there though or should I remove it entirely?
also I think I'd prefer if you landed on this view rather than on the list to open the chat for yourself

**Comment by hesterbruikman** [comment]
> also I think I'd prefer if you landed on this view rather than on the list to open the chat for yourself

+1

> Design reusing parts already found in Group Chats. @hesterbruikman we need something for the grey copy there though or should I remove it entirely?

I'd remove it. However in case of a community chat, we might show the channel description. Does that make sense? If so, can you add a variation in case of deeplink to a community chat

**Comment by flexsurfer** [comment]
as a first iteration, I would suggest doing next
* [ ] As Status core contributor (Marketing) I want to create a link for an influencer that directs users to a public chat, because they might drop off if they have to search for the chat 
* [ ] As a new user invited to a public chat on Status with a referral code, I want to join this public chat upon onboarding so I instantly have access to people who can help me navigate and discover the app
* [ ] Add public chat to list view when passed through deep link with referral code

**Comment by hesterbruikman** [comment]
>  Add public chat to list view when passed through deep link with referral code

I think this works, as long as the last two bullets are planned for. Shall I move those to a new issue and link under 'Future steps'?

It doesn't have the label, but I'd say it's a high prio feature as it directly addresses retention. The last two bullets should be addressed, but I can imagine wanting split up the work

**Comment by flexsurfer** [comment]
oh right, lets change the last one on  `Open public chat when passed through deep link with referral code` similar how 1-1 works
